README.md

```diff
@@ -21,6 +24,38 @@ Barnabas provides a way to run an [Apache Kafka][kafka] cluster on
 
 <!-- /TOC -->
 
+## Deploying Kubernetes/OpenShift cluster
+
+A Kafka/Zookeeper cluster can be deployed on a Kubernete or OpenShift cluster. So, first of all, such deployment is needed.
+
+### Kubernetes
+
+In order to interact with a Kubernetes cluster, be sure to have [the kuberctl tool][kubectl] installed.
```

Comment:
Typo ... `kuberctl` instead of `kubectl`.

Reply:
Fixed